src/test/resources/journeys.conf

```diff
@@ -15,52 +15,26 @@
 # Configure here your journeys. A journey is a sequence of requests at a certain load.
 
 journeys {
-
-  individual-journey = {
-    description = "Individual journey"
-    load = 9.1
-    parts = [
-        DigitalDisclosureIndividualJourney
-    ]
-  }
-
-  estate-journey = {
-    description = "Estate Journey"
-    load = 9.1
+  disclosure-company = {
+    description = "Submit a new disclosure (company)"
+    load = 1
     parts = [
-    DigitalDisclosureEstateJourney
+      auth-login,
+      hmrc-letter-reference,
+      company-details,
+      address-lookup,
+      personal-details,
+      address-lookup,
+      check-answers
     ]
   }
-
-  company-journey = {
-      description = "Company Journey"
-      load = 9.1
-      parts = [
-      DigitalDisclosureCompanyJourney
-      ]
-    }
-
-    limited-liability-partnership-journey = {
-          description = "Limited liability partnership Journey"
-          load = 9.1
-          parts = [
-          DigitalDisclosureLimitedLiabilityPartnershipJourney
-          ]
-        }
-
-    trust-journey = {
-          description = "Trust Journey"
-          load = 9.1
-          parts = [
-          DigitalDisclosureTrustJourney
-          ]
-        }
 }
 
-journeysToRun = [
-individual-journey,
-estate-journey,
-company-journey,
-limited-liability-partnership-journey,
-trust-journey
-]
\ No newline at end of file
+# Default behaviour is to run all journeys. If that is not what you need you can specify the list of journeys to run
+# journeysToRun = [
```

Comment:
The `journeysToRun` section runs all journeys by default (as per the comment in the template), no need to define it all journeys are to be run.